src/app.ts

```diff
@@ -63,6 +65,30 @@ const authenticatedRouter = new Router()
         ctx.state.toMember = midQuery.docs[0];
         return await next();
     })
+    .post('/api/members/:mid/request_invite', async ctx => {
+        const authUid = ctx.state.user.uid;
+        const authMember = await members.doc(authUid).get();
+        try {
+            const newOperation = {
+                creator_mid: authMember.get('mid'),
+                creator_uid: authUid,
+                op_code: 'REQUEST_INVITE',
+                data: {
+                    full_name: ctx.request.body.fullName,
+                    to_mid: ctx.state.toMember.get('mid'),
+                    to_uid: ctx.state.toMember.id,
+                    video_url: ctx.request.body.videoUrl,
+                },
+            };
+            await operations.add(newOperation);
```

Comment:
Eventually we want to move member creation code at https://github.com/rahafoundation/raha.io/blob/master/functions/index.js#L34 to here right?